src/engine/sync/enginesync.h

```diff
@@ -29,7 +29,7 @@ class EngineSync : public BaseSyncableListener {
     void requestBpmUpdate(Syncable* pSyncable, double bpm) override;
     void notifyInstantaneousBpmChanged(Syncable* pSyncable, double bpm) override;
     void notifyBeatDistanceChanged(Syncable* pSyncable, double beatDistance) override;
-    void notifyPlaying(Syncable* pSyncable, bool playing) override;
+    void notifyPlayingAudible(Syncable* pSyncable, bool playing) override;
```

Comment:
Please rename the parameters consistently, either all or none, even if the name doesn't matter. Applies multiple times.